applications/GeoMechanicsApplication/custom_workflows/custom_workflow_factory.cpp

```diff
@@ -0,0 +1,26 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_workflow_factory.h"
+#include "dgeosettlement.h"
+#include "custom_utilities/input_utility.h"
+#include <memory>
+
+namespace Kratos
+{
+
+KratosGeoSettlement* CustomWorkflowFactory::CreateKratosGeoSettlement()
+{
+    auto inputUtility = std::make_unique<InputUtility>();
+    return new KratosGeoSettlement(std::move(inputUtility));
```

Comment:
There is no need to use `std::move` if you would create a temporary:
```suggestion
    return new KratosGeoSettlement{std::make_unique<InputUtility>()};
```

Also, I prefer to use curly braces when constructing a new object.

Reply:
Done